.travis.yml

```diff
@@ -1,8 +1,12 @@
 language: python
 python:
   - "3.4"
+services:
+- postgresql
+before_script:
+  - psql -c 'create database evap;' -U postgres
 install:
-  - npm install -g less
+  - pip install psycopg2
```

Comment:
lets just put that into requirements.txt and remove it from here and the vagrant setup.